src/dali/plugin/input/rest/kraken.py

```diff
@@ -0,0 +1,468 @@
+# Copyright 2023 ndopencode
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kraken REST plugin links:
+# REST API: https://docs.kraken.com/rest/
+# Authentication: https://docs.kraken.com/rest/#section/Authentication
+# Endpoint: https://api.kraken.com
+
+# CCXT documentation:
+# https://docs.ccxt.com/en/latest/index.html
+
+import pytz
+import logging
+from datetime import datetime
+from typing import Dict, List, Optional, Union, Set
+
+from ccxt import Exchange, kraken
+
+from dali.abstract_ccxt_input_plugin import (
+    AbstractCcxtInputPlugin,
+)
+from dali.ccxt_pagination import (
+    AbstractPaginationDetailSet,
+)
+from dali.cache import (
+    load_from_cache,
+    save_to_cache
+)
+from dali.configuration import (
+    Keyword,
+    _FIAT_SET
+)
+from rp2.rp2_decimal import (
+    RP2Decimal,
+    ZERO
+)
+from rp2.logger import create_logger
+from dali.abstract_transaction import AbstractTransaction
```

Comment:
Did you run the full stack of for [Development Workflow](https://github.com/eprbell/dali-rp2/blob/main/README.dev.md#development-workflow)? It will probably sort out your imports.

The reason you don't get them is you might need to update pylint locally. Try:
`.venv/bin/pip3 install -e '.[dev]'`